packages/client/hmi-client/src/components/workflow/ops/simulate-ciemss/simulate-ciemss-operation.ts

```diff
@@ -13,7 +14,7 @@ export interface SimulateCiemssOperationState extends BaseState {
 	currentTimespan: TimeSpan;
 	numSamples: number;
 	solverStepSize: number;
-	method: string;
+	method: CiemssMethodOptions;
```

Comment:
Gorgeous!

Reply:
Should be fine, but just a reminder that `state` fields are intentionally kept very simple, because they needs to be able to be serialize into a json-string.